test/Elastic.Apm.Tests/ApiTests/ConvenientApiTransactionTests.cs

```diff
@@ -21,10 +22,19 @@ namespace Elastic.Apm.Tests.ApiTests
 	public class ConvenientApiTransactionTests
 	{
 		private const string ExceptionMessage = "Foo";
-		private const int SleepLength = 45;
+
 		private const string TransactionName = "ConvenientApiTest";
 		private const string TransactionType = "Test";
 
+		private int SleepLength
```

Comment:
Rather then re implementing this in each class maybe we can introduce a `Task.DelayMinimum()` extension method?

Reply:
I think there is no way to add static extension methods 🤔 

Nevertheless I agree that it'd make sense to implement this only once. I'll add a helper method.